### PR TITLE
Makingchanges

### PR DESCRIPTION
**Pull request by RhodaPickens**

### CODE REVIEW

**Comment by RhodaPickens** [comment]
I added my advice file, a separate stylesheet and a link within the index.html. When I synced with the upstream repository and then merged my branch with the master there was a merge conflict as someone else had added a link to the index.html in the same place as mine (the top). So I put mine below his and merged them.

File: index.html [hunk 1]
```diff
@@ -17,6 +17,7 @@ <h2>Advice for Newbies!</h2>
       <div class="all-advice-links">
         <div class="skillcrush">
           <a href="./advice/robbsky14-advice.html">Advice from Robbsky14</a><br />
```
Comment (RhodaPickens) [inline index.html]: This was the line that was conflicting with mine. I added mine below.

**Comment by redrambles** [comment]
Hey @RhodaPickens,

You did a great job and your code is all merged in! Yay!

Cheers,
Ann :)

**Comment by RhodaPickens** [comment]
Thank you!

**Pull request merged**